analyses/cms-open-data-ttbar/ttbar_analysis_pipeline.ipynb

```diff
@@ -60,7 +60,6 @@
     "from coffea.nanoevents import transforms\n",
```

Comment:
<p>Choose this updated description</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ekauffma/analysis-grand-challenge-emk/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>